aptos-move/block-executor/src/executor.rs

```diff
@@ -603,31 +631,34 @@ where
         let init_timer = VM_INIT_SECONDS.start_timer();
         let executor = E::init(executor_arguments);
         drop(init_timer);
+
         let data_map = UnsyncMap::new();
+        let mut unsync_view = LatestView::<T, S, X>::new_unsync_view(base_view, &data_map);
```

Comment:
What is difference between sync_view and unsync_view?

Reply:
one is for sequential execution (can use simple map and simple counter), and the other one is for parallel (using MVHashMap, atomic counter, capturing the reads, etc)